libs/libcommon/src/libcommon/processing_graph.py

```diff
@@ -585,17 +585,10 @@ def parse_id(id: str) -> tuple[str, str, Optional[str], Optional[str], str]:
         "difficulty": 70,
         "bonus_difficulty_if_dataset_is_big": 20,
     },
-    "split-duckdb-index-010": {
-        "input_type": "split",
-        "triggered_by": "config-parquet-metadata",
-        "job_runner_version": 1,
-        "difficulty": 70,
-        "bonus_difficulty_if_dataset_is_big": 20,
-    },
     "config-duckdb-index-size": {
         "input_type": "config",
-        "triggered_by": "split-duckdb-index",
-        "job_runner_version": 1,
+        "triggered_by": ["split-duckdb-index"],
```

Comment:
```suggestion
        "triggered_by": "split-duckdb-index",
```

a string is also valid. I find it more readable when we have only one entry

Reply:
Done, thanks!